static/js/condense.js

```diff
@@ -151,7 +151,7 @@ exports.show_message_expander = function (row) {
     }
 };
 
-exports.condense_and_collapse = function (elems) {
+exports.condense_and_collapse = function (elems) {  // This runs on page reload
```

Comment:
You should move this comment to a separate line.

Reply:
About that comment. This reload behavior is kind of user specific so I was wondering if I should keep it or remove it. I think a vague comment won't really help. But sure if you agree I will fix that